app/assets/stylesheets/finder_frontend.scss

```
@@ -25,6 +25,12 @@
   }
 }
 
+mark {
+  background-color: inherit;
+  color: inherit;
+  font-weight: bold;
```

```suggestion
  @include govuk-typography-weight-bold;
```
to use the mixin from govuk-frontend

---

Latest commit has this change made, thanks for the help!

---

still seeing font-weight: bold though